File: Data Structures & Algorithms/duplicate-integer/submission-5.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        const items = new Set();

        for (let x of nums) {
            items.add(x);
        }

        if (nums.length != items.size) {
            return true
        }

        return false
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-5.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        record = {}

        for num in nums:
            if record.get(num) is None:
                record.update({num: 1})
            else:
                record.update({num: record[num] + 1})
        sortedRecord = sorted(record.items(), key=lambda item: item[1], reverse=True)

        return [item[0] for item in sortedRecord][0:k]